### ISSUE TITLE
Avoid reading from file if value starts with '@'

### ISSUE THREAD

**Issue by markus1189** [jpmens/jo#42]
Is there any way to avoid the interpretation of the '@' value, which assumes the subsequent string is a file to read from?

I want to construct a json object with `@timestamp` as the value like this:

```
{"key": "@timestamp"}
```

via 

```
jo key="@timestamp"
```

but I found no way to escape the '@' such that it does **not** try to read from the `timestamp` file which does not exist

**Comment by jpmens** [comment]
Your example uses double quotes which are stripped by the shell. If you quote that again, it should work:

```bash
$ jo key='"@timestamp"'
{"key":"@timestamp"}
```

**Comment by markus1189** [comment]
That does the trick. Obvious now that you mention it ;)

**Comment by alewisohn** [comment]
This is now producing s string with escaped double quotes.
```
$ jo key='"@timestamp"'
{"key":"\"@timestamp\""}
```

**Comment by jpmens** [comment]
Fixed via #103
